static/templates/subscription_settings.hbs

```diff
@@ -9,15 +9,11 @@
               is_web_public=is_web_public }}
             <div class="stream-name">
                 <span title="{{name}}" class="stream-name-editable editable-section" >{{name}}</span>
-                {{#if can_change_name_description}}
-                <span class="editable" data-make-editable=".stream-name-editable"></span>
+                <span class="editable" data-make-editable=".stream-name-editable" {{#unless can_change_name_description}}style="display: none"{{/unless}}></span>
                 <span class="checkmark" data-finish-editing=".stream-name-editable">✓</span>
-                {{/if}}
             </div>
             <div class="button-group">
-                {{#if is_admin}}
-                <button class="button small rounded btn-danger deactivate" type="button" name="delete_button" title="{{t 'Delete stream'}}"> <i class="fa fa-trash-o" aria-hidden="true"></i></button>
-                {{/if}}
+                <button class="button small rounded btn-danger deactivate" type="button" name="delete_button" title="{{t 'Delete stream'}}" {{#unless can_delete_stream}}style="display: none"{{/unless}}> <i class="fa fa-trash-o" aria-hidden="true"></i></button>
```

Comment:
I recall I had a review in one of my PRs or maybe somewhere that we refrain from using inline CSS and use it when there is no other option (we use it in emails for obvious reasons). Can we use `hide` class here?

Reply:
I will change this.